vmdb/lib/miq_automation_engine/service_models/mixins/miq_ae_service_miq_request_mixin.rb

```diff
@@ -13,6 +13,7 @@ def get_option_last(key)
   end
 
   def user_message=(msg)
+    @object.update_attribute(:message, msg)
```

Comment:
Use `update_attributes(:message => msg)`so active record does not skip validation checks.